doc/object-creation.md

```diff
@@ -0,0 +1,53 @@
+# Admin object's management - create an object
+
+## Usage
+> Elements of the app are managed threw objects in *Admin*.
```

Comment:
Threw from throw, like a ball. I think you want to use through, like through the day

Reply:
true though - corrected